### DISCUSSION TITLE
Reload compile issues

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#663] [Ideas]
After compile issues have initially been loaded into the source, if I fix a line by inserting one or more lines, the issues seem to stay at their original ordinal line number and don't associate with the new line number.  Is there tan action that can reload the issues as we have with clearing them?

**Comment by worksofliam** [comment]
There is actually an enhancement I can make that will make the error relative and will move as the content changes. 

Will post back here when that's released.

**Comment by m-tyler** [comment]
If it's too much work I don't mind triggering an action to just reload the issues.
